modules/nw-pod-network-connectivity-implementation.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+//
+
+[id="nw-pod-network-connectivity-implementation_{context}"]
+= Implementation of connection health checks
+
+The connectivity check controller orchestrates connectivity verification checks in a {product-title} cluster. The results for the connection tests are stored in `PodNetworkConnectivity` objects in the `openshift-network-diagnostics` namespace. Connection tests are performed every minute in parallel.
+
+The Cluster Network Operator (CNO) deploys several resources to the cluster to send and receive connectivity health checks:
+
+Health check source:: This program deploys in single pod replica set managed by a `Deployment` object. The program consumes `PodNetworkConnectivity` objects and connects to the `spec.targetEndpoint` specified in each object.
+
+Health check target:: A pod deployed as part of a daemon set on every node in the cluster that listens for inbound health checks. The presence of this pod on every node allows for the testing of connectivity to each node.
```

Comment:
s/A pod deployed as part of a daemon set on every node in the cluster that listens for inbound health checks/A pod that listens for inbound health checks, which is deployed as part of a daemon set on every node in the cluster ??
Just to clarify what is doing the listening, the pod, the node, the cluster?

Reply:
@mburke5678, I'll refactor into several sentences to reduce the confusion. Thanks!